services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/rpc/_disk_usage.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import FastAPI
+from models_library.api_schemas_dynamic_sidecar.telemetry import DiskUsage
+from servicelib.rabbitmq import RPCRouter
+
+from ...modules.system_monitor import get_disk_usage_monitor
+
+router = RPCRouter()
+
+
+@router.expose()
```

Comment:
What kind of exceptions this function might raise? Right now it indicates None

Reply:
This is correct, it does not raise any exception, nor it raises any exception that needs to be handled by the user of the function.